### DISCUSSION TITLE
Anthem URLs embedded in the index file (dec version)

### DISCUSSION THREAD

**Discussion by moodytheboss** [CMSgov/price-transparency-guide#636] [Q&A] [answered]
The new Anthem index file seems to embed incorrect in-network urls. Is there anyone encounter the same problem? For example, if you try to download the data using the following url, you will be prompted with an error message:

https://anthembcbsnh.mrf.bcbs.com/2022-12_362_52S0_in-network-rates_01
_of_40.json.gz?&Expires=1674069079&Signature=bSyOEgWNCcAhOmXVmxLRtx8DU~CQpfaUU-s3pnEIfCbcx6P4lkXgP0TP9qQL48FYxt7UOBHbBBhXeEukYP3ByPDBBpa1V6r
zMIYnaAWcNzd4v3KCNdSuy6Rf46Vni4eE0XkP2HBzojr5uI6MAQqWi8wgB4GA9wsuhVO~XC5AIp9OM7AnGB4xCJ43Mx0NiDnyhZ-99v00s1qMVFCFgAEEzaMfPt7F41t7XcquoZM14IQ
gpSXlC4592C648tEeDsDB5BY6AHZliK3AVpBzAEXAGnF4mwyCRPH6O4vOnLX8Zi2QhQPYwGoUW-QZFHhF2QWF-keMD9ePe1FPsu6-rMNgqQ__&Key-Pair-Id=K27TQMT39R1C8A

The url is taken from the head of the following index file. Also the size of index file again back to 32Gb, which was 7Gb last month. 

{"reporting_entity_name":"Anthem Inc",
"reporting_entity_type":"health insurance issuer",
"reporting_structure":[
{"reporting_plans":[{"plan_name":"THE PERT CONSORTIUM","plan_id_type":"EIN","plan_id":"810741681","plan_market_type":"group"}],"in_network_f
iles":[{"description":"Independence BC : Personal Choice","location":"https://anthembcbsnh.mrf.bcbs.com/2022-12_362_52S0_in-network-rates_01
_of_40.json.gz?&Expires=1674069079&Signature=bSyOEgWNCcAhOmXVmxLRtx8DU~CQpfaUU-s3pnEIfCbcx6P4lkXgP0TP9qQL48FYxt7UOBHbBBhXeEukYP3ByPDBBpa1V6r
zMIYnaAWcNzd4v3KCNdSuy6Rf46Vni4eE0XkP2HBzojr5uI6MAQqWi8wgB4GA9wsuhVO~XC5AIp9OM7AnGB4xCJ43Mx0NiDnyhZ-99v00s1qMVFCFgAEEzaMfPt7F41t7XcquoZM14IQ
gpSXlC4592C648tEeDsDB5BY6AHZliK3AVpBzAEXAGnF4mwyCRPH6O4vOnLX8Zi2QhQPYwGoUW-QZFHhF2QWF-keMD9ePe1FPsu6-rMNgqQ__&Key-Pair-Id=K27TQMT39R1C8A"},{
"description":"BCBS Minnesota : Aware","location":"https://anthembcbsnh.mrf.bcbs.com/2022-12_720_27B0_in-network-rates_20_of_75.json.gz?&Exp
ires=1674069079&Signature=43LVRWCe5xyfJOny5N4J3oiLmmbyin8TtSAsz9KgXuzLqOE3-vEiMfh~hF6vsAmnS4iHp73i44welkIMcXc4EoN1JrHKhZgDZkCSSqCVqYUQ5NLHUN
s63JZDTZQ9RPrc8cegNlEWal0B29~BcNOcPelStMmyEEgsP0hGgjlzyRQTkrdETdTbnoNKvN-KqXg-Ehkz6pmt093iL~2Zs3~8dwGNoq3Q43DnxXHy06lmt2vzy0m7ZIsbYlvX8tJMWS
eGJWzPu-tP57xzW86Drs7xTbEjZ2wXI4qwmESMHGn8ShaVNDsA7AE53txMpmFahzTb2tfqkQzUJbXrzqJaaW436Q__&Key-Pair-Id=K27TQMT39R1C8A"},{"description":"High
mark BCBS WV : Super Blue Plus","location":"https://anthembcbsnh.mrf.bcbs.com/2022-12_443_69B0_in-network-rates_18_of_19.json.gz?&Expires=16
74069079&Signature=fmixHLwSJDGUHYo4QDwQF9q66GTAsjTEBBBRtjE5zL5s86UmZmUBRJjx9Cc47hWKg34g10ne0KXZk59owXCN8loMXRgdsZwOoFVXbCfqoz~yiO5A17PZEJTJc
MxSfHZRXv~NxhfYwpBfCrzGGZqk9kw9Rw~~0HXwemWkTuFdPCQF7t-yaDIGz-uGcnklBtgIXrjHaSDjEY6gUFw2mQkMBV~fVDgPE8W5k-eL-xMhU3qL4diyXz20EnMci8EDwG-Xp5LxZ
5WtQJLNZ36a2JeJKBRBdypsssBRe6bjqcWn41g2yN174LGcWYCnjkzFquRtOSAlgJesJTsj7dNTUNOcIw__&Key-Pair-Id=K27TQMT39R1C8A"},

**Comment by KDDE37** [comment, marked as answer]
It looks like you may be missing (~) from the url which is causing the error.  Files are able to be downloaded and opened without issue. File size is at 15.7 GB. not seeing 32GB.
